tests/coverage/coverage.cpp

```diff
@@ -336,6 +336,26 @@ TEST(Coverage, analyser)
     analyser->analyseModel(model);
 }
 
+TEST(Coverage, analyserConvertToInt)
+{
+    auto analyser = libcellml::Analyser::create();
+    auto parser = libcellml::Parser::create();
+    auto model = parser->parseModel(fileContents("coverage/converttoint.cellml"));
+
+    auto printer = libcellml::Printer::create();
```

Comment:
Not used.

Reply:
[db70d31](https://github.com/cellml/libcellml/pull/1126/commits/db70d31c7247e2eeb3fc76ce169bad38fb65b500)